viral/ensemble_reactivation.py

```
@@ -1218,13 +1256,272 @@ def multiple_sessions() -> None:
+    )
+    # shaded_line_plot(
+    #     response_pre,
+    #     x_axis=np.arange(-30, 30) / 30,
+    #     color="orange",
+    #     label="preactivation",
+    # )
+    plt.xlabel("Time (s) from event onset")
+    1 / 0
+
+
+def compare_run_results(W_py: np.ndarray, W_mat: np.ndarray) -> None:
```

Ruff (0.14.0)

Redefinition of unused `compare_run_results` from line 1109

(F811)